uavcan/register/384.Access.1.0.uavcan

```diff
@@ -64,19 +64,68 @@
-#   - P - persistent, V - volatile
-#
-#   Name pattern        | Flags | Type      | Purpose
-# ----------------------+-------+-----------+--------------------------------------------------------------------------
-#   uavcan.node_id      | MP    | uint16    | Contains the node-ID of the local node. Values above the maximum valid
-#                       |       |           | node-ID for the current transport indicate that the node-ID is not set;
-#                       |       |           | if plug-and-play is supported, it will be used by the node to obtain an
-#                       |       |           | automatic node-ID. Invalid values other than 65535 should be avoided for
-#                       |       |           | consistency. The factory-default value should be 65535.
-# ----------------------+-------+-----------+--------------------------------------------------------------------------
+# common functions. These conventions are not mandatory to follow, but implementers are recommended to adhere because
+# they enable enhanced introspection capabilities and simplify device configuration and diagnostics.
+#
+#   REGISTER NAME PATTERN                             TYPE            FLAGS                   RECOMMENDED DEFAULT
+# =====================================================================================================================
+#
+#   uavcan.node.id                                      natural16       mutable, persistent     65535 (unset/PnP)
+#
+# Contains the node-ID of the local node. Values above the maximum valid node-ID for the current transport
+# indicate that the node-ID is not set; if plug-and-play is supported, it will be used by the node to obtain an
+# automatic node-ID. Invalid values other than 65535 should be avoided for consistency.
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan.node.description                             string          mutable, persistent     (empty)
+#
+# Human-readable verbose description of this specific node defined by the integrator (not vendor).
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan.cookie                                       string          mutable, persistent     (empty)
+#
+# Arbitrary data stored by an external autoconfiguration authority (e.g., a mission computer, an auto-configuration
+# utility, etc.). This data is never to be used by the node itself; instead, the node merely provides raw data
+# storage space.
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan.pub.PORT_NAME.id                             natural16       mutable, persistent     65535 (unset, invalid)
+#   uavcan.sub.PORT_NAME.id                             ...             ...                     ...
```

Comment:
seeing as [the ditto mark](https://en.wikipedia.org/wiki/Ditto_mark) goes back to 934BC we probably shouldn't try to change it now.